content/docs/alerting/configuration.md

```diff
@@ -462,10 +462,12 @@ endpoint:
   "externalURL": <string>,  // backling to the Alertmanager.
   "alerts": [
     {
+      "status": "<resolved|firing>",
       "labels": <object>,
       "annotations": <object>,
       "startsAt": "<rfc3339>",
-      "endsAt": "<rfc3339>"
+      "endsAt": "<rfc3339>",
+      "generatorURL": "string", // backling to the prometheus graph
```

Comment:
I hope can supporting a parameter to modify the ip of the generatorURL, because if Prometheus running in kubernetes it will need another address be used by receiver.